.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -179,7 +179,7 @@ jobs:
 
       - uses: actions/upload-artifact@v3
         if: |
-          github.event_name == 'release' ||
+          github.event_name == 'pull_request' ||
```

Comment:
Do we want this change? Shouldn't it create the wheel on release too?
Or you expect to change it back? I'd add a new branch to be removed instead in this case.

Reply:
This is to download & validate the wheels before uploading to PyPI. We change it back on the release branch.